voltixApp/VoltixApp/Model/UTXO/Blockchair/Blockchair.swift

```diff
@@ -41,44 +41,19 @@ class Blockchair: Codable {
 		var type: String?
 		var scriptHex: String?
 		var balance: Int?
-		var balanceUsd: Double?
-		var received: Int?
-		var receivedUsd: Double?
-		var spent: Int?
-		var spentUsd: Double?
-		var outputCount: Int?
-		var unspentOutputCount: Int?
-		var firstSeenReceiving: String?
-		var lastSeenReceiving: String?
-		var firstSeenSpending: String?
-		var lastSeenSpending: String?
-		var scripthashType: String?
-		var transactionCount: Int?
-		
-		
+
 		var balanceInBTC: String {
 			formatAsBitcoin(balance ?? 0)
 		}
+        
+        func balanceInFiat(balance: Double, price: Double, includeCurrencySymbol: Bool = true) -> String{
```

Comment:
looks like `includeCurrencySymbol` is not used anymore

Reply:
Thanks